service/controller/aws/v7/resource/configmap/resource.go

```diff
@@ -38,15 +46,25 @@ func New(config Config) (*Resource, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.Logger must not be empty", config)
 	}
 
+	if config.ClusterIPRange == "" {
+		return nil, microerror.Maskf(invalidConfigError, "%T.ClusterIPRange must not be empty", config)
+	}
```

Comment:
Logic here is the same. Not all flags are required because we use Calico differently on Azure.